Refit/XmlContentSerializer.cs

```diff
@@ -26,12 +26,14 @@ public Task<HttpContent> SerializeAsync<T>(T item)
         {
             var xmlSerializer = new XmlSerializer(item.GetType(), settings.XmlAttributeOverrides);
 
-            using (var output = new StringWriter())
+            using (var stream = new MemoryStream())
             {
-                using (var writer = XmlWriter.Create(output, settings.XmlReaderWriterSettings.WriterSettings))
+                using (var writer = XmlWriter.Create(stream, settings.XmlReaderWriterSettings.WriterSettings))
                 {
+                    var encoding = settings.XmlReaderWriterSettings.WriterSettings?.Encoding ?? Encoding.Unicode;
```

Comment:
What should the default be? Unicode or UTF8? Any reason for Unicode over UTF8 as the default?

Reply:
StringWriter default encoding is Unicode, so I keep it